docs/interpreter/cassandra.md

```diff
@@ -789,5 +811,7 @@ Below are the configuration parameters and their default value.
 [standard CQL syntax]: http://docs.datastax.com/en/cql/3.1/cql/cql_using/use_collections_c.html
 [Tuple CQL syntax]: http://docs.datastax.com/en/cql/3.1/cql/cql_reference/tupleType.html
 [UDT CQL syntax]: http://docs.datastax.com/en/cql/3.1/cql/cql_using/cqlUseUDT.html
+[Zeppelin dynamic form]: http://zeppelin.incubator.apache.org/docs/latest/manual/dynamicform.html
```

Comment:
This one also need to be changed to `http://zeppelin.apache.org/docs/latest/manual/dynamicform.html`.